demoapps/src/main/content/jcr_root/apps/dantademo/components/page/home/.content.xml

```diff
@@ -2,6 +2,6 @@
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
     jcr:primaryType="cq:Component"
     jcr:title="Danta Demo - Home"
-    sling:resourceSuperType="danta/components/page/htmlbase"
+    sling:resourceSuperType="dantademo/components/page/htmlbase"
```

Comment:
I think this change may be correct, but please review.  This changed the supertype of this component (it was `danta`, not `dantaaemdemo` before).  Since dantademo/htmlbase extends danta/htmlbase, I think the new value is correct.

Reply:
Yeah, I did this because there needs to be a component called 'htmlbase' as a page rendering base component in order for the global dialog to work.